adapters/tuzlanski.js

```diff
@@ -0,0 +1,131 @@
+'use strict';
+
+import { REQUEST_TIMEOUT } from '../lib/constants';
+import { default as baseRequest } from 'request';
+const request = baseRequest.defaults({timeout: REQUEST_TIMEOUT});
+import cheerio from 'cheerio';
+import Coordinates from 'coordinate-parser';
```

Comment:
What do you think about using http://turfjs.org/docs/#point for this? We're already using Turf in fetch (https://github.com/openaq/openaq-api/blob/develop/package.json#L41) and would be good to not add another library into the mix.